tests/deepsparse/pipelines/test_pipeline.py

```diff
@@ -114,30 +114,34 @@ def test_pipeline_executor_num_workers():
 
 @mock_engine(rng_seed=0)
 def test_pipeline_call_is_async(engine_mock):
-    # here we make engine_forward take a different amount of time
-    # based on the input. the total time should be the longest
-    # forward call, instead of sum.
-    #
-    # The time each forward takes is just summing up character counts
-    # in each word. So "aaaa" will sleep for 50ms whereas "a" will sleep for 10ms
+    # attempts to verify that pipeline calls to engine are async
+    # by executing at timing test with different number of workers
+
     executor = ThreadPoolExecutor(max_workers=1)
     pipeline = Pipeline.create("token_classification", batch_size=1, executor=executor)
 
-    def sleep_for_10ms(xs):
-        time.sleep(10 / 1000)
+    def sleepy_engine(xs):
+        # each call to engine_forward also sleeps
+        time.sleep(20 / 1000)
         return pipeline.engine(xs)
```

Comment:
just noticed this, intentional? issue with mocking?
```suggestion
        return pipeline.engine_forward(xs)
```

Reply:
Intentional - this sleep_engine function replaces engine_forward